ctapipe/containers.py

```diff
@@ -572,6 +586,23 @@ class SimulationConfigContainer(Container):
     corsika_high_E_detail = Field(
         nan, "More details on high E interaction model (version etc.)"
     )
+    tracking_mode = Field(
+        TrackingMode.UNKNOWN, "Tracking Mode (see docs of `TrackingMode`)"
+    )
```

Comment:
Might be better to have this all in a SchedulingBlock container, which can be common to both Simulations and Observations.

Reply:
This is directly from simtel... It's more part of the simtel data model than the CTA one.

Reply:
By why hard-code things to the way SimTel does it? My point is we should have a common definition of pointing information, and then use it for SimTel, some other simulation, or real Observations.